loader/loader.c

```diff
@@ -5437,6 +5443,17 @@ VKAPI_ATTR VkResult VKAPI_CALL terminator_CreateInstance(const VkInstanceCreateI
                 icd_create_info.enabledExtensionCount++;
             }
         }
+#ifdef LOADER_ENABLE_LINUX_SORT
+        // Force on "VK_KHR_get_physical_device_properties2" for Linux as we use it for GPU sorting.
+        if (icd_term->scanned_icd->api_version < VK_API_VERSION_1_1) {
+            prop = get_extension_property(VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME, &icd_exts);
```

Comment:
Just to be robust, what happens if the driver doesn't support the extension?

Reply:
Changes for this coming up.  I now will only do this if at least one device supports vkGetPhysicalDeviceProperties2(KHR).  Otherwise, I skip the sorting.

Reply:
FYI, that's because I need the PCI bus information to search the PCI directory for the default GPU.  (If you wanted to know).